hi-ml-cpath/src/health_cpath/utils/report_utils.py

```diff
@@ -139,13 +148,16 @@ def get_hyperdrive_metrics_table(metrics_df: pd.DataFrame, metrics_list: Sequenc
         values: pd.Series = metrics_df.loc[metric]
         mean = values.mean()
         std = values.std()
-        row = [metric] + [f"{v:.3f}" for v in values] + [f"{mean:.3f} ± {std:.3f}"]
+        round_values: List[str] = [f"{v:.3f}" if v is not None else f"{v}" for v in values]
```

Comment:
[Minor] Here I presume `f"{v}"` evaluates to `"None"` for the missing values, right? How about replacing this with a literal, like "N/A", "–", "(missing)", ...?

Reply:
Yes, it will be None. Now replaced by "N/A"